src/Codeception/Configuration.php

```diff
@@ -646,7 +646,7 @@ private static function prepareParams($settings)
             }
 
             // .env and ini files
-            if (preg_match('~(\.ini|\.env)$~', $paramStorage)) {
+            if (preg_match('~(\.ini$|\.env)~', $paramStorage)) {
```

Comment:
ok that makes sense regarding `.ini`.
I'm no regex wizard but I noticed a problem with your updated pattern though: https://regex101.com/r/hO5rU9/2

It's now matching stuff like `.envy` or anything with `.env` in it.